### DISCUSSION TITLE
Add new theme from external url in Marp VS Code

### DISCUSSION THREAD

**Discussion by Theskrtnerd** [marp-team/marp#338] [Q&A] [answered]

**Comment by yhatt** [comment, marked as answer]
:arrow_right: https://github.com/marp-team/marp-vscode#use-custom-theme-css-%EF%B8%8F

![](https://user-images.githubusercontent.com/3993388/185166310-4d0ef984-3e62-4793-ac6d-c8ad1e6d8104.png)
